samples/Java/src/com/microsoft/azure/servicebus/samples/BasicSendReceiveWithQueueClient.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.samples;
+
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+public class BasicSendReceiveWithQueueClient {
+    // Connection String for the namespace can be obtained from the Azure portal under the
+    // 'Shared Access policies' section.
+    private static final String connectionString = "{connection string}";
+    private static final String queueName = "{queue name}";
+    private static IQueueClient queueClient;
+    private static int totalSend = 100;
+    private static int totalReceived = 0;
+
+    public static void main(String[] args) throws Exception {
+
+        Log.log("Starting BasicSendReceiveWithQueueClient sample");
+
+        // create client
+        Log.log("Create queue client.");
+        queueClient = new QueueClient(new ConnectionStringBuilder(connectionString, queueName), ReceiveMode.PeekLock);
+
+        // send and receive
+        queueClient.registerMessageHandler(new MessageHandler(queueClient), new MessageHandlerOptions(1, false, Duration.ofMinutes(1)));
+        for (int i = 0; i < totalSend; i++) {
+            int j = i;
+            Log.log("Sending message #%d.", j);
```

Comment:
>Log.log("Sending message #%d.", j); [](start = 12, length = 35)

Does log.log always log to screen... If so sending 100 messages there will be 200 messages on screen.. do we want to just do one message per send and may be send 10 or 15 should be good enough?